####    SIMULATION PARAMETERS ####
hierachical_ems: 
    OutlierRatio: 0.9650408061736067                # prior outlier probability [0, 1) (default: 0.1)
    MaxIterationEM: 18                              # maximum number of EM iterations (default: 20)
    ToleranceEM: 0.0046787892375589855              # absolute tolerance of EM (default: 1e-3)
    RelativeToleranceEM: 0.8673860505442554         # relative tolerance of EM (default: 1e-1)
    MaxOptiIterations: 2                            # maximum number of optimization iterations per M (default: 2)
    Sigma: 0.3                                      # 0.3 initial sigma^2 (default: 0 - auto generate)
    MaxiSwitch: 2                                  # maximum number of switches allowed (default: 2)
    AdaptiveUpperBound: False                        # Introduce adaptive upper bound to restrict the volume of SQ (default: false)
    Rescale: True                                   # normalize the input point cloud (default: true)
    MaxLayer: 2                                     # maximum depth
    Eps: 0.013130820860359358                       # 0.03 IMPORTANT: varies based on the size of the input pointcoud (DBScan parameter)
    MinPoints: 100                                  # default: 500 DBScan parameter required minimum points
    Max_superquadrics: 4                            # DBScan parameter required minimum points



#### REAL WORLD PARAMETERS ####
# hierachical_ems: 
#     OutlierRatio: 0.9955271167239488                # prior outlier probability [0, 1) (default: 0.1)
#     MaxIterationEM: 18                              # maximum number of EM iterations (default: 20)
#     ToleranceEM: 0.004304036224804307              # absolute tolerance of EM (default: 1e-3)
#     RelativeToleranceEM: 0.803576732836785         # relative tolerance of EM (default: 1e-1)
#     MaxOptiIterations: 2                            # maximum number of optimization iterations per M (default: 2)
#     Sigma: 0.3                                      # 0.3 initial sigma^2 (default: 0 - auto generate)
#     MaxiSwitch: 2                                  # maximum number of switches allowed (default: 2)
#     AdaptiveUpperBound: True                        # Introduce adaptive upper bound to restrict the volume of SQ (default: false)
#     Rescale: True                                   # normalize the input point cloud (default: true)
#     MaxLayer: 4                                     # maximum depth
#     Eps: 0.03968163127120671                       # 0.03 IMPORTANT: varies based on the size of the input pointcoud (DBScan parameter)
#     MinPoints: 150                                  # default: 500 DBScan parameter required minimum points
#     Max_superquadrics: 4                            # DBScan parameter required minimum points